src/PrestaShopBundle/Entity/Repository/OrderInvoiceRepository.php

```diff
@@ -61,18 +61,15 @@ public function __construct(Connection $connection, $tablePrefix)
      */
     public function countByOrderState(array $shopIds)
     {
-        $sql = <<<'SQL'
-SELECT COUNT(o.id_order) AS nbOrders, o.current_state as id_order_state
-FROM `{table_prefix}order_invoice` oi
-LEFT JOIN `{table_prefix}orders` o ON oi.id_order = o.id_order
-WHERE o.id_shop IN(:shopIds)
-AND oi.number > 0
-GROUP BY o.current_state
-SQL;
+        $sql = 'SELECT COUNT(o.id_order) AS nbOrders, o.current_state as id_order_state
+            FROM `{table_prefix}order_invoice` oi
+            LEFT JOIN `{table_prefix}orders` o ON oi.id_order = o.id_order
```

Comment:
```suggestion
            INNER JOIN `{table_prefix}orders` o ON oi.id_order = o.id_order
```

Reply:
That changes the behavior of the query. Isn't it ?

Reply:
Ping @PierreRambaud

Reply:
since the selection is made on the joined table only, I believe it should not change the result of this function

Reply:
It's because of `o.id_shop` in the WHERE condition, so the `o` alias is mandatory, and because it's mandatory, you have to use INNER for better performance.

Reply:
Thanks for explanation @PierreRambaud . I approve the request of @PierreRambaud